internal/command/delta.go

```diff
@@ -70,26 +72,45 @@ func delta(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("preparing new deployment: %w", err)
 	}
 
-	// Create a new deployment delta
-	//
-	log.Print("Creating a new deployment delta...\n")
 	client, err := api.NewClient(apiUrl, apiToken, http.DefaultClient)
 	if err != nil {
 		return err
 	}
-	res, err := client.CreateDelta(context.Background(), orgID, appID, delta)
-	if err != nil {
-		return err
+
+	var res *ht.DeploymentDelta
+	if deltaUseId == "" {
+
+		log.Print("Creating a new deployment delta...\n")
+		res, err = client.CreateDelta(context.Background(), orgID, appID, delta)
+		if err != nil {
+			return err
+		}
+		res.Metadata.Url = fmt.Sprintf("%s/orgs/%s/apps/%s/envs/%s/draft/%s", uiUrl, orgID, appID, delta.Metadata.EnvID, res.ID)
+
+	} else {
+		log.Print("Retrieving existing delta...\n")
+		existing, err := client.GetDelta(context.Background(), orgID, appID, deltaUseId)
```

Comment:
Do we really need to fetch existent delta? Why not to send an update (`PATCH`) request straight away?

Reply:
I had thought it might improve the UX a bit here. But since the deltas API returns a 404 for patch unknown id anyway, this doesn't actually give us much.